lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3376,6 +3399,29 @@ namespace {
                                 std::nullopt);
     }
 
+    bool isNormalCxxMemberMethod(const clang::CXXMethodDecl *method) {
+      // Check if the method is a constructor or destructor
+      if (llvm::dyn_cast<clang::CXXConstructorDecl>(method) ||
+          llvm::dyn_cast<clang::CXXDestructorDecl>(method)) {
+        return false;
+      }
+
+      // Exclude compiler-generated, special, and trivial functions
+      return !method->isOverloadedOperator() && // Not an operator overload
```

Comment:
I think in many of these cases the comments do not add additional information compared to the code.